tools/docker/builder/openwrt/Dockerfile

```diff
@@ -44,12 +44,17 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
 
 WORKDIR /home/openwrt/openwrt
 
+COPY --chown=openwrt:openwrt configs/* /home/openwrt/
```

Comment:
nack.
Avoid using globs at all cost when using COPY, because any change in the copied file will trigger a new build from here.
If you have any file you don't want lying in `configs`, it's also going to be included in the image.